coding/browser-support.md

```diff
@@ -0,0 +1,36 @@
+# Browser support
+
+Our baseline browser support for our websites. There are two levels of support. 
+
+## Full support
+
+Any browser with full support needs to be pixel perfect and all functionality 
+should work.
+
+| Browser | Version |
+| ------- | ------- |
+| Chrome | 55 or greater |
+| Firefox | 52 or greater |
+| Safari | 11 or greater |
+| IE | 11 or greater |
+| Edge | 15 or greater |
+| Opera | 48 or greater |
+
+## Accessible support
+
+The content on these browsers should be readable but my not look perfect.
```

Comment:
"The content on these browsers should be readable but do **not have to** look perfect." might be clearer?

Reply:
`does not`? Isn't `content` singular?

Reply:
...yes. Yes it is.
`The content on these browsers should be readable but does not have to look perfect.`